src/control/settings/Settings.cpp

```diff
@@ -980,6 +985,7 @@ void Settings::save()
 		saveProperty((const char*) "scrollbarHideType", "none", root);
 	}
 
+	WRITE_BOOL_PROP(disableScrollbarFadeout);
```

Comment:
Needs to be moved up or down... currently it is between the scrollbarHideType setting and its comment line.  If you look in the ~/.xournalpp/settings.xml file you'll see:

```
 ...
  <property name="scrollbarHideType" value="none"/>
  <property name="disableScrollbarFadeout" value="true"/>
  <!--Hides scroolbars in the main window, allowed values: "none", "horizontal", "vertical", "both"-->
...
```